doc/string/inspect.rdoc

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
Nit: why is there a blank line on the first line?

Reply:
Removed.